internal/common/ingest/ingestion_pipeline.go

```diff
@@ -206,7 +207,7 @@ func (ingester *IngestionPipeline[T]) Run(ctx context.Context) error {
 			} else {
 				for _, msgId := range msg.GetMessageIDs() {
 					util.RetryUntilSuccess(
-						context.Background(),
+						armadacontext.Background(),
```

Comment:
The fact we create a new Background context here looks wrong but wasn't introduced in this PR.  I'll take a look at changing this in a follow up.